Makefile

```diff
@@ -1,6 +1,7 @@
 VERSION ?= $(shell ./get-versions.sh VERSION)
 REVISION ?= $(shell ./get-versions.sh REVISION)
 DIST ?= unstable
+DIST_VERSION ?= "~"$(shell sed '/CODENAME/!d;s/^.*=//' /etc/lsb-release)"1"
```

Comment:
is there any place where the system understands that lsb-release is a dependency to parse this? lsb-release is not included by default everywhere